starters/react-ts/src/client/index.html

```diff
@@ -0,0 +1,2 @@
+<script type="module" src="/$app/mount.js">
```

Comment:
I don't see a file named "mount" in this PR. Does it exist?

Reply:
It's provided by `@fastify/react`'s virtual modules.